edgexfoundry/edgex-examples
# feat: Migrate Camera Management example to V3

closes #194







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions
run non-secure EdgeX stack
run make build
run ./app-camera-management -cp -d -o
Verify service starts without error.
